src/main/java/net/fabricmc/loader/discovery/RuntimeModRemapper.java

```diff
@@ -162,14 +164,16 @@
 
 	private static byte[] remapAccessWidener(byte[] input, Remapper remapper) {
 		try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(input), StandardCharsets.UTF_8))) {
-			AccessWidener accessWidener = new AccessWidener(FabricLoader.INSTANCE);
-			accessWidener.read(bufferedReader, null);
+			AccessWidener accessWidener = new AccessWidener();
+			AccessWidenerReader accessWidenerReader = new AccessWidenerReader(accessWidener);
+			accessWidenerReader.read(bufferedReader);
```

Comment:
This should really be passing in `"intermediary"` too given the remapper is renaming from that. If the access widener isn't using intermediary names chances are it won't be applying properly because of that. Also makes FabricMC/access-widener#3 fixable without breaking anything.